src/main/java/com/epam/ta/reportportal/ws/controller/impl/LaunchController.java

```diff
@@ -229,6 +229,27 @@ public LaunchResource getLaunch(@PathVariable String projectName, @PathVariable
 	}
 
 	@Override
+	@RequestMapping(value = "/latest", method = GET)
+    @ResponseBody
+    @ResponseStatus(OK)
+    @ApiOperation("Get latest unique launches for specified project by filter")
+	public Iterable<LaunchResource> getLatestLaunches(@PathVariable  String projectName, @FilterFor(Launch.class) Filter filter,
+            @SortFor(Launch.class) Pageable pageable) {
+		return getLaunchMessageHandler.getLatestLaunches(normalizeId(projectName), filter, pageable);
+	}
+
+	@Override
+    @RequestMapping(value = "/latest/mode", method = GET)
```

Comment:
they have different access level
in debug mode only project members have control